doc/nrf/ug_thread_commissioning.rst

```diff
@@ -209,6 +202,182 @@ See the following pages in the `OpenThread CLI Reference`_ on GitHub for an over
 * `Commissioner CLI commands`_
 * `Joiner CLI commands`_
 
```

Comment:
Please move this whole section, including the separator, to the very end of the file.

Reply:
fixed